refinery/data_set_manager/views.py

```diff
@@ -363,21 +361,17 @@ def post(self, request, *args, **kwargs):
                             import_file.delay(file_store_item.uuid)
 
                 if request.is_ajax():
-                    return HttpResponse(
-                        json.dumps({
-                            'success': 'Data set imported',
-                            'data': {'new_data_set_uuid': dataset_uuid}
-                        }),
-                        content_type='application/json'
-                    )
+                    return JsonResponse({
+                        'success': 'Data set imported',
+                        'data': {'new_data_set_uuid': dataset_uuid}
+                    })
                 return HttpResponseRedirect(
                     reverse(self.success_view_name, args=[dataset_uuid])
                 )
             else:
                 error = 'Problem parsing ISA-Tab file'
                 if request.is_ajax():
-                    return HttpResponse(json.dumps({'error': error}),
-                                        content_type='application/json')
+                    return JsonResponse({'error': error})
```

Comment:
Would an HTTP error response be appropriate here? (But that's not something you've changed, so perhaps better not to wade into that.)